packages/headless/src/controllers/commerce/facets/core/generator/headless-commerce-facet-generator.test.ts

```diff
@@ -36,17 +46,17 @@ describe('CommerceFacetGenerator', () => {
           })
         );
         break;
-      case 'regular':
-      case 'dateRange': // TODO
-      case 'hierarchical': // TODO
-      default:
+      case 'dateRange':
         facets.push(
-          buildMockCommerceRegularFacetResponse({
+          buildMockCommerceDateFacetResponse({
             facetId,
-            field: 'some_regular_field',
+            field: 'some_date_field',
           })
         );
         break;
+      case 'hierarchical': // TODO
+      default:
+        break;
```

Comment:
leaning towards category for commerce, and reserve hierarchical for the hierarchical of old

Reply:
This is modeled after the values we're getting from the Search API, though